### PR TITLE
Setup storage device for DesktopGL

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

File: MonoGame.Framework/Storage/StorageContainer.cs [hunk 1]
```diff
@@ -118,9 +119,15 @@ internal StorageContainer(StorageDevice device, string name, PlayerIndex? player
...
 			var saved = "";
-#elif LINUX || MONOMAC
+#elif MONOMAC
             // We already have a SaveData folder on Mac/Linux.
             var saved = StorageDevice.StorageRoot;
+#elif DESKTOPGL
+            string saved = "";
+            if(CurrentPlatform.OS == OS.Linux)
+                saved = StorageDevice.StorageRoot;
+            else
```
Comment (tomspilman) [inline MonoGame.Framework/Storage/StorageContainer.cs]: How about we do something like this...
```
else if (CurrentPlatform.OS == OS.Windows)
{}
else
   throw new Exception("Unexpected platform!");
```
... or something so we get a good runtime error when the platform detection fails to work.

@KonajuGames @dellis1972  ?

Comment (KonajuGames) [inline MonoGame.Framework/Storage/StorageContainer.cs]: That makes sense to me.

Comment (dellis1972) [inline MonoGame.Framework/Storage/StorageContainer.cs]: Good call

-----Original Message-----

Comment (cra0zy) [inline MonoGame.Framework/Storage/StorageContainer.cs]: Done as you've asked.

**Comment by tomspilman** [comment]
Thanks... merging!

**Pull request merged**